packages/js/src/plugins/auctionHouseModule/models/Bid.ts

```diff
@@ -17,84 +17,97 @@ import { AuctionHouse } from './AuctionHouse';
 import { Nft, NftWithToken, Sft, SftWithToken } from '../../nftModule';
 
 /** @group Models */
-export type Bid = Readonly<
-  {
-    /** A model identifier to distinguish models in the SDK. */
-    model: 'bid';
+export type Bid = Readonly<PublicBid | PrivateBid>;
```

Comment:
Can you revert that change? This will not play nicely with our typedocs.

If you want explicit types for `PublicBid` and `PrivateBid`, could you do something like this?

```ts
export type PublicBid = Bid & { isPublic: true };
export type PrivateBid = Bid & { isPublic: false };
```